jbpm-runtime-manager/src/main/java/org/jbpm/runtime/manager/impl/PerRequestRuntimeManager.java

```diff
@@ -131,7 +131,8 @@ public void validate(KieSession ksession, Context<?> context) throws IllegalStat
     @Override
     public void disposeRuntimeEngine(RuntimeEngine runtime) {
     	if (isClosed()) {
-    		throw new IllegalStateException("Runtime manager " + identifier + " is already closed");
+            logger.warn("Runtime manager {} is already closed", identifier);
```

Comment:
A new `logger` member should be created in this class belonging to the same - using `logger` from AbstractRuntimeManager might lead to confusion in logs, and might not be displayed in case a different logger level is set for that class.

Reply:
Done, thanks for your review @afalhambra !